core/src/test/java/org/infinispan/replication/ForceSyncAsyncFlagsTest.java

```diff
@@ -61,34 +61,7 @@ public void testForceAsyncFlagUsage() throws Exception {
       // verify FORCE_ASYNCHRONOUS flag on SYNC cache
       cache1.withFlags(Flag.FORCE_ASYNCHRONOUS).put("k", "v");
       verify(mockTransport).invokeRemotely(anyCollectionOf(Address.class),
-                                           any(ReplicableCommand.class), eq(ResponseMode.ASYNCHRONOUS_WITH_SYNC_MARSHALLING), anyLong(),
-                                           any(ResponseFilter.class), any(DeliverOrder.class), anyBoolean());
-   }
-
-   public void testForceSyncFlagUsage() throws Exception {
```

Comment:
I think this test should stay, at least until we remove the FORCE_SYNCHRONOUS flag...